jdi-light-bdd-tests/src/test/resources/features/FileInput.feature

```diff
@@ -0,0 +1,23 @@
+Feature: File Input element functionality
+
+  Scenario: Upload file by enabled file input element
+    Given I open "Html5 Page"
+    Then "Avatar" label text equals to "Profile picture:"
+    And "Avatar" label text contains "picture"
+    And "Avatar" is enabled
+    And "Avatar" is displayed
+    And "Avatar" attribute "id" equals to "avatar"
+    When I upload file "/src/test/resources/general.xml" by "Avatar" file input element
+    Then "Avatar" text equals to "C:\fakepath\general.xml"
+    And "Avatar" text contains "general.xml"
+    And "Avatar" file input element value equals to "C:\fakepath\general.xml"
```

Comment:
replace with 
    And "Avatar" text equals to "C:\fakepath\general.xml"
    And "Avatar" text contains "general.xml"

Reply:
fixed